src/routes/committee/setlist/+page.svelte

```diff
@@ -0,0 +1,249 @@
+			<StructuredListRow>
+				<StructuredListCell>{song.name}</StructuredListCell>
+				<StructuredListCell>{song.artist}</StructuredListCell>
+				<StructuredListCell
+					>{@const validUrl = isValidUrl(song.link)}
+					<Button
+						href={validUrl ? song.link : undefined}
+						target="blank"
+						kind="ghost"
+						size="small"
+						iconDescription={validUrl ? song.link : 'Invalid URL'}
+						icon={PlayFilledAlt}
+						disabled={!validUrl}
+					/></StructuredListCell
+				>
+				<StructuredListCell>
+					{#each Object.entries(data.musiciansForSongs) as [key, value], j}
+						{#if key == song.id}
+							{#each value as musician}
+								{musician[0]} - {musician[1]}
+								<Button
+									size="small"
+									kind="ghost"
+									class="removePerson"
+									on:click={() => {
+										selectedSong = i
+										selectedUser = musician[0]
+										selectedInstrument = musician[1]
+										selectedUserEntry = j
+										openDelUser = true
+									}}>X</Button
+								>
+								<br />
+							{/each}
+						{/if}
+					{/each}
+				</StructuredListCell>
+				<StructuredListCell>
+					<Button
+						kind="ghost"
+						size="small"
+						iconDescription="Add musician"
+						icon={UserFollow}
+						on:click={() => {
+							selectedSong = i
+							openAdd = true
+						}}
+					/>
+					<Button
+						kind="danger-tertiary"
+						size="small"
+						iconDescription="Delete"
+						icon={MusicRemove}
+						on:click={() => {
+							selectedSong = i
+							openDel = true
+						}}
+					/>
+				</StructuredListCell>
+			</StructuredListRow>
+		{/each}
+	</StructuredList>
+</Grid>
+
+<Modal
+	danger
+	modalHeading="Delete from setlist"
+	primaryButtonText="Delete"
+	primaryButtonIcon={MusicRemove}
+	secondaryButtonText="Cancel"
+	bind:open={openDel}
+	on:click:button--primary={() => {
+		RemoveSongFromSetlist()
+		openDel = false
+	}}
+	on:click:button--secondary={() => {
+		openDel = false
+	}}
+>
+	<p>Delete {data.songs[selectedSong]?.name}?</p>
+</Modal>
+
+<Modal
+	danger
+	modalHeading="Remove from song"
+	primaryButtonText="Remove"
+	primaryButtonIcon={MusicRemove}
+	secondaryButtonText="Cancel"
+	bind:open={openDelUser}
+	on:click:button--primary={() => {
+		RemoveParticipantFromSong()
+		openDelUser = false
+	}}
+	on:click:button--secondary={() => {
+		openDelUser = false
+	}}
+>
+	<p>Remove {selectedUser} from {data.songs[selectedSong]?.name}?</p>
+</Modal>
+
+<Modal
+	bind:open={openAdd}
+	modalHeading="Login"
+	primaryButtonText="Confirm"
+	on:click:button--primary={() => {
+		AddParticipantToSong()
+		openAdd = false
+	}}
+	hasScrollingContent
+>
+	<Form>
+		<ComboBox
+			titleText="Participant"
+			bind:selectedId={participant}
+			items={participantListItems}
+			required
+		/>
+		<br />
+		<TextInput bind:value={instrument} required labelText="Instrument" />
+	</Form>
+	<br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /><br
+	/><br />
```

Comment:
Wat on earth is dit

Reply:
Zorgen dat de modal lang is anders krijg je een 4 px hoge combobox selectie, t is echt kut